docs/_includes/head/custom.html

```diff
@@ -0,0 +1,5 @@
+<link rel="apple-touch-icon" sizes="180x180" href="/assets/images/favicon/apple-touch-icon.png">
```

Comment:
Does this file to add the favicon feature?

Can we split this PR to different PR to let us review the codes easier?

Reply:
Next time I will notice that rule.